src/client/conn/http1.rs

```diff
@@ -255,6 +255,10 @@ where
 {
     type Output = crate::Result<()>;
 
+    /// # Note
+    ///
+    /// Attempting to poll a Connection after it has been dropped may lead
```

Comment:
does it though?

you should not use term undefined behavior lightly

Reply:
https://doc.rust-lang.org/std/future/trait.Future.html. It says that once a Future is resolved, calling poll might result in panic, indefinite blocking, or other problems. Should I revise the documentation to say that?

Reply:
Well, technically you cannot poll `Connection` in safe manner after it has been dropped :)
You should consider `Connection` in context of `SendRequest` as you use them together

Reply:
In that case should I document the `handshake` method as well?

Reply:
I think `conn` module documentation would be good place:
https://github.com/hyperium/hyper/blob/master/src/client/conn/mod.rs#L1

Reply:
I have pushed some changes to the conn, http1, and http2 modules. I appreciate any feedback :)